tests/xfstests/partition.pm

```diff
@@ -332,8 +338,23 @@ sub run {
     my $filesystem = get_required_var('XFSTESTS');
     my %para;
     if (check_var('XFSTESTS', 'nfs')) {
-        install_dependencies_nfs;
-        setup_nfs_server("$NFS_VERSION");
+        if (get_var('XFSTESTS_NFS_SERVER')) {
+            install_dependencies_nfs;
+            server_configure_network($self);
+            setup_nfs_server("$NFS_VERSION");
+            mutex_create('xfstests_nfs_server_ready');
+            wait_for_children;
+        }
+        elsif (get_var('PARALLEL_WITH')) {
+            setup_static_mm_network('10.0.2.102/24');
+            assert_script_run('mkdir -p /nfs/test /nfs/scratch');
+            $NFS_SERVER_IP = '10.0.2.101';
```

Comment:
Having these IP set as perl constants somewhere instead of copy paste them would help searching them / understanding they are related.